app/src/main/java/com/p2p/presentation/home/games/GamesViewModelFactory.kt

```diff
@@ -0,0 +1,14 @@
+package com.p2p.presentation.home.games
+
+import android.content.Context
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.p2p.data.UserSession
+import com.p2p.framework.SharedPreferencesUserInfoStorage
+
+class GamesViewModelFactory(private val context: Context) : ViewModelProvider.Factory {
```

Comment:
No entendía qué verga era esto así que dejo esto por aquí:
`https://medium.com/koderlabs/viewmodel-with-viewmodelprovider-factory-the-creator-of-viewmodel-8fabfec1aa4f`